pkg/runtime/framework/plugins/mpi/mpi.go

```diff
@@ -56,6 +59,11 @@ var _ framework.EnforceMLPolicyPlugin = (*MPI)(nil)
 var _ framework.WatchExtensionPlugin = (*MPI)(nil)
 var _ framework.ComponentBuilderPlugin = (*MPI)(nil)
 
+var (
+	errorFailedToBuildConfigMap = fmt.Errorf("failed to build hostfile configMap")
+	errorFailedToBuildSecret    = fmt.Errorf("failed to build SSH Auth secret")
+)
```

Comment:
Why do you want to create vars for these 2 errors if you don't use them in other places (e.g. unit test)

Reply:
Good catch. I was thinking of implementing UTs, but I found there are no corner cases based on our programming errors. Let's remove those.

Reply:
Done.